### PR TITLE
chatbot

### PR DESCRIPTION
**Pull request by TrentonWalkup**
script to run LS chatbot via API and verify responses via CSV.

### CODE REVIEW

**Comment by mikepoe** [comment]
LGTM

**Comment by Bradern8t0r** [comment]
LGTM

File: demo/chatbot/test.csv [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿INTENT,INPUT,OUTPUT1,Hello,"Hola, soy Erin, su representante automatizado de LegalShield. Le puedo contestar sus preguntas de nuestros planes, ayudarle con su cuenta, o contestar muchas questionies de legal comun."2,What is LegalShield,"LegalShield ha proporcionado planes de servicio legal a familias de Norteamérica en los 50 estados y 4 provincias de Canadá durante más de 40 años. Nuestros clientes pueden obtener asesoría jurídica y consejos de abogados calificados con sólo llamar a un número gratuito y usar nuestra aplicación móvil LegalShield para Apple o Android. Hay muchos otros beneficios incluidos, pero aún en el nivel más básico, nuestros planes te permiten estar tranquilo. Conoce más en https://www.legalshield.com."
```
Comment (TrentonWalkup) [inline demo/chatbot/test.csv]: idk what format the actual csv will be in.

File: demo/chatbot/main.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
-var apiai = require('apiai');
-var secure = require('../secret.js');
-var assert = require('assert');
-var app = apiai(chatbot_api_token);
-var csv = require('csv-parser')
-var fs = require('fs');
-var read = fs.createReadStream;
+const apiai = require('apiai');
```
Comment (TrentonWalkup) [inline demo/chatbot/main.js]: doing some research. we need to move to a place where we start using const vs var

**Pull request merged**